src/main/java/com/nfl/glitr/registry/datafetcher/query/OverrideDataFetcher.java

```diff
@@ -54,22 +56,57 @@ public Object get(DataFetchingEnvironment environment) {
         try {
             return overrideMethod.invoke(obj, environment);
         } catch (InvocationTargetException e) {
+            Throwable targetException = e.getTargetException();
 
             // If the override method threw a RuntimeException just send it up
-            if (e.getTargetException() instanceof RuntimeException) {
-                logger.error("Something went wrong - Unable to fetch result for overrideMethod={{}} of {}", overrideMethod.getName(), obj.getClass().getSimpleName(), e.getTargetException());
-                throw (RuntimeException) e.getTargetException();
+            if (targetException instanceof RuntimeException) {
+                log(overrideMethod.getName(), obj.getClass().getSimpleName(), targetException);
+                throw (RuntimeException) targetException;
             }
 
             // Otherwise, wrap it up in a Glitr Exception
-            logger.error("Something went wrong - Unable to fetch result for overrideMethod={{}} of {}", overrideMethod.getName(), obj.getClass().getSimpleName());
-            throw new GlitrException("Overwrite method exception", e.getTargetException());
+            log(overrideMethod.getName(), obj.getClass().getSimpleName(), targetException);
+            throw new GlitrException("Overwrite method exception", targetException);
         } catch (Exception e) {
-            logger.error("Something went wrong - Unable to fetch result for overrideMethod={{}} of {}", overrideMethod.getName(), obj.getClass().getSimpleName());
+            log(overrideMethod.getName(), obj.getClass().getSimpleName(), e);
         }
         return null;
     }
 
+    private void log(String methodName, String className, Throwable targetException) {
+        Object[] args = new Object[targetException instanceof RuntimeException ? 3 : 2];
+        args[0] = methodName;
+        args[1] = className;
+        if (targetException instanceof RuntimeException) {
+            args[2] = targetException instanceof GlitrOverrideException ? targetException.getCause() : targetException;
```

Comment:
Hello @omurzin 
Maybe it will be a good idea to replace magic numbers with constants with meaningful name?